### PR TITLE
A few minor tweaks

### PR DESCRIPTION
**Pull request by sjn**
Hi,

I shaved some yaks; hope it's appreciated.

### CODE REVIEW

File: twirssi.pl [hunk 1]
```diff
@@ -1925,7 +1926,7 @@ sub get_updates_child {
                 unless (&get_timeline( $fh, $frusers[ $fix_replies_index{$username} ],
                                                $username, $twits{$username}, \%context_cache, $is_regular )) {
                     $error++;
-                    push @error_types, 'replies';
+                    push @error_types, 'Failed to get_timeline(frusers)';
```
Comment (zigdon) [inline twirssi.pl]: @frusers perhaps?

File: twirssi.pl [hunk 2]
```diff
@@ -2012,7 +2013,7 @@ sub get_updates_child {
                 print $fh "t:block ac:$username nick:$_ epoch:$show_blocks->{$_}\n";
             }
         }
-        next if $error > $errors_beforehand;
+         next if scalar @error > $errors_beforehand;
```
Comment (zigdon) [inline twirssi.pl]: fix indent please

File: twirssi.pl [hunk 3]
```diff
@@ -1904,18 +1904,19 @@ sub get_updates_child {
 
     my $error = 0;
```
Comment (zigdon) [inline twirssi.pl]: Do we still need this? It's just `scalar @error`, no?

File: twirssi.pl [hunk 4]
```diff
@@ -322,10 +322,10 @@ sub format_expand {
     my %args = @_;
     $args{fmt} =~ s/\$n/\@$args{nick}/g;
     if (defined $args{data} and $args{data} ne '') {
-        $args{fmt} =~ s/\${|\$}//g;
+        $args{fmt} =~ s/\$\{|\$\}//g;
```
Comment (zigdon) [inline twirssi.pl]: I don't mind escaping the `{`, but I don't think they have any special meaning except when they contain digits?

Comment (sjn) [inline twirssi.pl]: Dan Boger said:
> 
> In twirssi.pl:
> 
> > @@ -322,10 +322,10 @@ sub format_expand {
> >      my %args = @_;
> >      $args{fmt} =~ s/\$n/\@$args{nick}/g;
> >      if (defined $args{data} and $args{data} ne '') {
> > -        $args{fmt} =~ s/\${|\$}//g;
> > +        $args{fmt} =~ s/\$\{|\$\}//g;
> 
> I don't mind escaping the {, but I don't think they have any special meaning
> except when they contain digits?

You get a warning with newer perls if it isn't escaped. Not entirely sure 
why (haven't bothered to look, just wanted to get rid of the noise).


- Salve

-- 
#!/usr/bin/env perl
sub AUTOLOAD{$AUTOLOAD=~/.*::(\d+)/;seek(DATA,$1,0);print# Salve Joshua Nilsen
getc DATA}$"="'};&{'";@_=unpack("C*",unpack("u*",':50,$'.#    <sjn@foo.no>
'3!=0"59,6!`%%P\0!1)46%!F.Q`%01,`'."\n"));eval "&{'@_'}";  __END__ is near! :)

File: twirssi.pl (hunk 1 above)
Comment (sjn) [inline twirssi.pl]: Dan Boger said:
> 
> In twirssi.pl:
> 
> > @@ -1925,7 +1926,7 @@ sub get_updates_child {
> >                  unless (&get_timeline( $fh, $frusers[ $fix_replies_index{
> $username} ],
> >                                                 $username, $twits{$usernam
> e}, \%context_cache, $is_regular )) {
> >                      $error++;
> > -                    push @error_types, 'replies';
> > +                    push @error_types, 'Failed to get_timeline(frusers)';
> 
> @frusers perhaps?

My thought was to make the different error messages distinct enough to figure 
out which error was triggered. Dunno if the contents of @frusers is useful in 
that error message. What do you think?


- Salve

-- 
#!/usr/bin/env perl
sub AUTOLOAD{$AUTOLOAD=~/.*::(\d+)/;seek(DATA,$1,0);print# Salve Joshua Nilsen
getc DATA}$"="'};&{'";@_=unpack("C*",unpack("u*",':50,$'.#    <sjn@foo.no>
'3!=0"59,6!`%%P\0!1)46%!F.Q`%01,`'."\n"));eval "&{'@_'}";  __END__ is near! :)

File: twirssi.pl [hunk 5]
```diff
@@ -1935,44 +1936,44 @@ sub get_updates_child {
                       "ac:$username\n";
             }
         }
-        next if $error > $errors_beforehand;
+        next if scalar @error > $errors_beforehand;
```
Comment (gedge) [inline twirssi.pl]: remove the word `scalar` - putting `@error` in a scalar context (next to the `>`) is sufficient

File: twirssi.pl (hunk 1 above)
Comment (gedge) [inline twirssi.pl]: My original plan for `@error_types` was to be a succinct list of _areas_ of errors, not English error messages, as evidenced by the last usage of the var (see line with `For recurring DMs errors`)

**Pull request closed without merging**